src/framework/uicomponents/qml/Muse/UiComponents/internal/StyledMenuItem.qml

```diff
@@ -202,7 +202,7 @@ ListItemBlank {
             id: secondaryIconLabel
             Layout.alignment: Qt.AlignLeft
             Layout.preferredWidth: 16
-            color: root.modelData?.iconColor ?? ui.theme.fontPrimaryColor
+            color: Boolean(root.modelData) && Boolean(root.modelData.iconColor) ? root.modelData.iconColor : ui.theme.fontPrimaryColor
```

Comment:
I'd be interested if changing ?? into || also fixed the problem, but it's fine.

Reply:
ok, || works, let's use it